tests/ui/pages/desktop/details.py

```diff
@@ -0,0 +1,32 @@
+from pypom import Region
+from selenium.webdriver.common.by import By
+
+from base import Base
+
+
+class Details(Base):
+    """ Details page. """
+
+    def wait_for_page_to_load(self):
+        self.wait.until(lambda _: self.header.is_displayed)
+        return self
+
+    @property
+    def header(self):
+        return self.DescriptionHeader(self)
+
+    class DescriptionHeader(Region):
+        """
+        Represents the header of the detail page.
+        """
+        _root_locator = (By.CLASS_NAME, 'addon-description-header')
+
+        _name_locator = (By.TAG_NAME, 'h1')
+
+        @property
+        def is_displayed(self):
+            return self.name.is_displayed()
+
+        @property
+        def name(self):
+            return self.find_element(*self._name_locator)
```

Comment:
As with collections, page objects should not return web elements. I suspect you want to return the text contents of this element instead.